src/pages/repository/components/tabs/deployments-tab/index.tsx

```diff
@@ -0,0 +1,87 @@
+import formatDistanceToNow from 'date-fns/formatDistanceToNow'
+import { FiExternalLink } from 'react-icons/fi'
+import { GrDeploy } from 'react-icons/gr'
+import { IoIosCheckmarkCircle } from 'react-icons/io'
+import { useNavigate } from 'react-router-dom'
+
+import { shortenAddress } from '@/helpers/shortenAddress'
+import { useGlobalStore } from '@/stores/globalStore'
+
+export default function DeploymentsTab() {
+  const navigate = useNavigate()
+  const [userRepo] = useGlobalStore((state) => [state.repoCoreState.selectedRepo.repo])
+  const deployments = [...(userRepo?.deployments ?? [])].reverse()
+
+  function gotoUser(userAddress: string) {
+    if (userAddress) {
+      navigate(`/user/${userAddress}`)
+    }
+  }
+
+  if (deployments.length === 0) {
+    return (
+      <div className="h-full w-full px-2 flex flex-col">
+        <div className="flex flex-col w-full border-gray-300 border-[1px] rounded-lg bg-white overflow-hidden">
+          <div className="rounded-t-lg flex justify-between bg-gray-200 border-b-[1px] border-gray-300 items-center gap-2 py-2 px-4 h-12"></div>
+
+          <div className="rounded-b-lg w-full bg-white text-liberty-dark-100 overflow-hidden">
+            <div className="flex flex-col gap-2 h-32 w-full items-center justify-center">
+              <GrDeploy className="h-7 w-7" />
+              <h1 className="text-lg font-medium">Looks like this repository hasn't any deployments yet.</h1>
```

Comment:
Looks like this repository has no previous deployments.

lets use the above message?